### PR TITLE
Document published entity endpoints

### PR DESCRIPTION
**Pull request by trodrigues**

### CODE REVIEW

File: apiary/cma.apib [hunk 1]
```diff
@@ -201,6 +201,24 @@ When using this endpoint, an ID will be automatically generated for the created
 
     + Attributes (Content Type)
 
+## Published Content Type Collection [/spaces/{space_id}/public/content_types?access_token={access_token}]
```
Comment (grncdr) [inline apiary/cma.apib]: The `access_token` parameter should not be here, (it should also be removed on line 170, my bad)

File: apiary/cma.apib [hunk 2]
```diff
@@ -201,6 +201,24 @@ When using this endpoint, an ID will be automatically generated for the created
...
+
++ Parameters
+    + space_id: 5smsq22uwt4m (required, string) - ID of the Space in form of a string
+
+### Get all published content types of a Space [GET]
+
+Retrieves only Content Types which have been published.
```
Comment (grncdr) [inline apiary/cma.apib]: Reword this in all cases to say "Retrieves the published versions of `$whatever`", it should be very clear that you are getting a different state than what the "normal" endpoints might return.

Comment (trodrigues) [inline apiary/cma.apib]: :+1:

**Comment by grncdr** [comment]
:+1:

**Pull request merged**